### ISSUE TITLE
Advanced Search "level warning" callout fails WCAG AA contrast

### ISSUE THREAD

**Issue by tnajdek** [zotero/zotero#5990]
The warning in Advanced Search uses `--accent-red` text on an `--accent-red10` background, which does not meet the WCAG AA contrast minimum of **4.5:1** for normal-size text.

<img width="987" height="172" alt="Image" src="https://github.com/user-attachments/assets/25f73831-1738-42be-b179-423a792d3bc0" />

**Measured contrast:**
  - Light theme: ~**4.07:1**
  - Dark theme: ~**2.1:1**

_(Because of the transparency used, exact values vary slightly depending on the pane background, but they stay under 4.5:1.)_

I've used similar-looking, but WCAG AA compliant values for a similar error message in the template validation (#5989):

https://github.com/zotero/zotero/pull/5989/changes#diff-93c6953905b34b9e6ab7df1ac8062dbc789ceaa9f9656cd0d4a9411762ec8323

I suggest we promote these (or perhaps we can come up with even better pair of colors) to global variables and make them the standard for error messages like this.

**Comment by tnajdek** [comment]
Light is now #b81d29 on #fce8ea, 5.5:1 
 Dark is now #ff7078 on #40191c, 5.7:1

I've also removed `--accent-red10` since it had no users.
